pkg/kubelet/apis/podresources/server_v1_test.go

```diff
@@ -141,10 +160,12 @@ func TestListPodResourcesV1(t *testing.T) {
 			m.On("GetPods").Return(tc.pods)
 			m.On("GetDevices", string(podUID), containerName).Return(tc.devices)
 			m.On("GetCPUs", string(podUID), containerName).Return(tc.cpus)
+			m.On("GetMemory", string(podUID), containerName).Return(tc.memory)
 			m.On("UpdateAllocatedDevices").Return()
 			m.On("GetAllocatableCPUs").Return(cpuset.CPUSet{})
 			m.On("GetAllocatableDevices").Return(devicemanager.NewResourceDeviceInstances())
-			server := NewV1PodResourcesServer(m, m, m)
+			m.On("GetAllocatableMemory").Return([]state.Block{})
```

Comment:
why empty? Don't we need testcase(s) on which we return valid allocatable memory?

Reply:
It the List tests, so for `GetAllocatableMemory` we just set an empty state. I do not sure if it necessary at all to set it here, but I just followed the device and CPU manager examples.
```
m.On("GetAllocatableCPUs").Return(cpuset.CPUSet{})
m.On("GetAllocatableDevices").Return(devicemanager.NewResourceDeviceInstances())
```

Reply:
I guess we can start like this and improve coverage later - applies indeed to the other resource types.